packages/web-app-user-management/src/views/Users.vue

```diff
@@ -356,7 +303,7 @@ export default defineComponent({
     ...mapMutations('runtime/spaces', ['UPDATE_SPACE_FIELD']),
 
     calculateListHeaderPosition() {
-      this.listHeaderPosition = this.$refs.appBar.getBoundingClientRect().height
+      this.listHeaderPosition = this.$refs.template.$refs.appBar.getBoundingClientRect().height
```

Comment:
Same as for groups, soft-fails in unit tests because this.$refs.template.$refs is undefined in the unit tests of the view (-> gives an ugly error message in the unit tests)